wit/i2c.wit

```diff
@@ -0,0 +1,82 @@
+package wasi:i2c@0.1.0;
```

Comment:
The version here and in the other files should be "0.2.0-draft", following [this advice](https://github.com/WebAssembly/WASI/issues/583).